packages/charts/react-charting/src/components/DonutChart/DonutChart.types.ts

```diff
@@ -69,6 +69,12 @@ export interface IDonutChartProps extends ICartesianChartProps {
    * @default false
    */
   roundCorners?: boolean;
+
+  /**
+   * Prop to enable the multiple legend selection
+   * @default false
+   */
+  canSelectMultipleLegends?: boolean;
```

Comment:
>canSelectMultipleLegends?: boolean; [](http://example.com/codeflow?start=2&length=35)

not needed. This is an existing property of the legends control

Reply:
done